### DISCUSSION TITLE
Add output of --debug to templates

### DISCUSSION THREAD

**Discussion by huornlmj** [aquasecurity/trivy#5624] [Ideas]
### Description

I would like to add the -d flag to show specifically what items are being ignored so that I can include this ignore info the templated output.  I don't know if this is possible or not.

### Target

None

### Scanner

None

**Comment by simar7** [comment]
Is this about misconfiguration scanning? If so, we've added support for that here https://github.com/aquasecurity/trivy/pull/5550 and will be out in the next release.

**Reply by huornlmj** [reply]
Actually @simar7 it's across all modules and is a general feature request, so it wouldn't just be for misconfiguration.
